sdk/cosmos/tests.yml

```diff
@@ -244,4 +244,23 @@ jobs:
         goals: verify
         options: '$(ProfileFlag) -Dgpg.skip $(AdditionalArgs) -pl $(ProjectList)'
 
-      TestResultsFiles: '**/junitreports/TEST-*.xml'
\ No newline at end of file
+      TestResultsFiles: '**/junitreports/TEST-*.xml'
+
+  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
```

Comment:
This template is in wrong tests.yml

We cannot club `azure-cosmos` live tests and `azure-spring-data-cosmosdb` live tests. 

We should create a separate `tests.yml` in `sdk/cosmos/azure-spring-data-cosmosdb` directory which will use its own `test-resources.json` file which you have created above.

Reply:
Hi, I have added a new tests.yml in the directory sdk/cosmos/azure-spring-data-cosmosdb. By the way, what is the concern about why azure-cosmos and azure-spring-data-cosmosdb live tests cannot be put in one tests.yml? Will this mean that these two tests would always be triggered out together?

Reply:
Yes, you are right, if we add them in the same file, then they will always run together, which we don't want. 
We want separate CIs for both of these projects.

Reply:
Why would we create a separate ```tests.yml``` file. Shouldn't the goal be to make sure these live tests run along with the Cosmos live tests? The whole point of our path structure is that things that are co-located is that they build and ship (roughly) together.

Reply:
Yea, this was older comment from me, please disregard this - we have just 1 tests.yml file now.